sklearn/linear_model/_bayes.py

```diff
@@ -489,18 +464,6 @@ class ARDRegression(RegressorMixin, LinearModel):
         to false, no intercept will be used in calculations
         (i.e. data is expected to be centered).
 
-    normalize : bool, default=False
-        This parameter is ignored when ``fit_intercept`` is set to False.
-        If True, the regressors X will be normalized before regression by
-        subtracting the mean and dividing by the l2-norm.
-        If you wish to standardize, please use
-        :class:`~sklearn.preprocessing.StandardScaler` before calling ``fit``
-        on an estimator with ``normalize=False``.
-
-        .. deprecated:: 1.0
-            ``normalize`` was deprecated in version 1.0 and will be removed in
-            1.2.
-
```

Comment:
`X_offset_` and `X_scale_` further down in the docstring reference `normalize=True`. I think they need updating, like you did for `LinearRegression`.

Reply:
good catch, thanks